src/Enums/EventSubType.php

```diff
@@ -11,15 +11,15 @@ class EventSubType
     // Triggers whenever a user updates their account.
     public const USER_UPDATE = 'user.update';
 
-    // Triggers whenever a user follows another user.
-    public const USER_FOLLOW = 'user.follow';
-
```

Comment:
This would be a breaking change although it is technically not working.
I suggest only changing the value to `channel.follow` and adding a deprecated PHPDoc attribute to allow migration until removed.

```php
/**
 * @deprecated Please use the CHANNEL_FOLLOW constant
 */
public const USER_FOLLOW = 'channel.follow';

```

Reply:
You're right! I applied this.